test/lit/help/wasm-fuzz-types.test

```diff
@@ -0,0 +1,29 @@
+;; RUN: wasm-fuzz-types --help | filecheck %s
```

Comment:
Looks like this was added as the auto-updater emits it but it was not checked in? Sgtm.